test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -28,6 +28,18 @@ def test_database_exists_returns_true_when_the_database_exists
         assert ActiveRecord::ConnectionAdapters::CockroachDBAdapter.database_exists?(db_config.configuration_hash),
           "expected database #{db_config.database} to exist"
       end
+
+      def test_using_follower_reads_connects_properly
+        database_config = { "use_follower_reads": true, "adapter" => "cockroachdb", "database" => "activerecord_unittest" }
```

Comment:
can this be called `use_follower_reads_for_type_introspection`? (or another name you like?) i want it to be more clear that it doesn't affect all queries

also, could you make a "configuration" section in the README that documents this setting, and any others we add later